ocaml/xapi/xapi_ha.mli

```diff
@@ -0,0 +1,116 @@
+(*
+ * Copyright (C) 2017 Citrix Systems Inc.
+ *
+ * This program is free software; you can redistribute it and/or modify
+ * it under the terms of the GNU Lesser General Public License as published
+ * by the Free Software Foundation; version 2.1 only. with the special
+ * exception on linking described in file LICENSE.
+ *
+ * This program is distributed in the hope that it will be useful,
+ * but WITHOUT ANY WARRANTY; without even the implied warranty of
+ * MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+ * GNU Lesser General Public License for more details.
+ *)
+
+(** Functions for implementing 'High Availability' (HA).
+    @group High Availability (HA) *)
+
+val ha_redo_log : Redo_log.redo_log
+(** The redo log instance used for HA *)
```

Comment:
I think it's safer to put it below the values, as the first comment is always associated with the module apparently, see http://caml.inria.fr/pub/docs/manual-ocaml/ocamldoc.html ("15.2.1  Placement of documentation comments").